tests/unit/dataactvalidator/test_scripts_loadFSRS.py

```diff
@@ -0,0 +1,73 @@
+from unittest.mock import Mock
+
+import pytest
+
+from dataactcore.models.fsrs import FSRSAward, FSRSSubaward
+from dataactvalidator.scripts import loadFSRS
+from tests.unit.dataactcore.factories import (
+    FSRSAwardFactory, FSRSSubawardFactory)
+
+
+@pytest.yield_fixture()
+def no_award_db(database):
```

Comment:
> do you think we should be mocking a database for the unit tests?

That's a good question; it's very tempting to use the existing test database since we'll use it in the validation tests, but that doesn't mean it's a good idea. For example, if I want to run just the tests around these models, I need to wait an extra few seconds for the database to be set up and tore down. Eh, I don't think the benefits outweigh the costs here -- I'm okay sticking with postgres unless someone objects.